uniprot-config/src/main/resources/search-fields-config/uniref-search-fields.json

```diff
@@ -120,24 +121,25 @@
   },
   {
     "seqNumber": 5,
-    "id": "created",
-    "fieldName": "created",
-    "label": "Date published",
+    "id": "date_modified",
+    "fieldName": "date_modified",
+    "label": "Date of last modification",
     "itemType": "SINGLE",
```

Comment:
I updated it to "Date of last modification" in UniRef because we do not have 2 types of modification like in UniProt. In UniProt we have "Date of last entry modification" and "Date of last sequence modification". This is why we have the extra word "entry".